runtime/doc/api.txt

```diff
@@ -2750,15 +2750,15 @@ nvim_buf_set_extmark({buffer}, {ns_id}, {line}, {col}, {opts})
                     buffer or end of the line respectively. Defaults to true.
                   • sign_text: string of length 1-2 used to display in the
                     sign column.
-                  • sign_hl_group: name of the highlight group used to
-                    highlight the sign column text.
-                  • number_hl_group: name of the highlight group used to
-                    highlight the number column.
-                  • line_hl_group: name of the highlight group used to
-                    highlight the whole line.
-                  • cursorline_hl_group: name of the highlight group used to
-                    highlight the sign column text when the cursor is on the
-                    same line as the mark and 'cursorline' is enabled.
+                  • sign_hl_group: highlight group used to highlight the sign
```

Comment:
repeating `highlight` sounds a bit unnattural, maybe just ` highlight group used for the sign column text`  (and so on for the others)